src/style-spec/reference/v8.json

```diff
@@ -2490,6 +2490,42 @@
           }
         }
       },
+      "round": {
+        "doc": "Rounds the input to the nearest integer. Halflway values are rounded away from zero. E.g., `[\"round\", -1.5]` evaluates to -2.",
```

Comment:
Nit: It’s somewhat unusual to begin a sentence with “E.g.” “For example” is more easily understood.